src/Jenssegers/Mongodb/Relations/BelongsTo.php

```diff
@@ -69,4 +72,29 @@ protected function whereInMethod(EloquentModel $model, $key)
     {
         return 'whereIn';
     }
+
+    /**
+     * @inheritdoc
```

Comment:
Remove this doc or fill actual info, please

Reply:
Done.